### DISCUSSION TITLE
[Question][Hw1] Inserting newline character in print operation

### DISCUSSION THREAD

**Discussion by park-sunho** [prosyslab-classroom/cs424-program-reasoning#382] [Q&A]
In `transfer` function in semantics.ml, when dealing with print operation, do we need to automatically print newline character after printing value on the screen? Or do we just need to print only the value?
(I checked if we don't print newline character then `make test` fails, but I wonder if it is the case even for hidden test cases.)

**Comment by KihongHeo** [comment]
Hi.

Please follow the `make test` results. They are provided to help you, NOT confuse you.
